docs/internals/contributing/writing-code/unit-tests.txt

```diff
@@ -401,6 +401,20 @@ test failures. You can adjust this behavior with the ``--parallel`` option::
 You can also use the ``DJANGO_TEST_PROCESSES`` environment variable for this
 purpose.
 
+Tests hang with "in progress in another thread" message
+-------------------------------------------------------
+
+On Mac OSX (High Sierra and newer versions), you might see this message while
```

Comment:
Prefer "macOS" (as used elsewhere in the docs).

Reply:
Thanks, updated.